examples/.vscode/launch.json

```diff
@@ -0,0 +1,188 @@
+{
```

Comment:
Hi! Just wonder why we need this file here? We probably can remove this if it's unnecessary:)